test/test_device_hydro_generation_constructors.jl

```diff
@@ -819,152 +832,3 @@ end
         end
     end
 end
-
```

Comment:
Many tests need to enabled and working not removed.

Reply:
Some of them are already present and one of the formulation is no longer valid